src/pudl/metadata/resources/pudl.py

```diff
@@ -102,8 +102,11 @@
         "sources": ["pudl"],
     },
     "out_ferc714__hourly_estimated_state_demand": {
-        "title": "Estimated Hourly State Electricity Demand",
-        "description": "Estimated hourly electricity demand for each state, scaled such that it matches the total electricity sales by state reported in EIA 861.",
+        "description": {
+            "additional_summary_text": (
+                "estimated hourly electricity demand for each state, scaled such that it matches the total electricity sales by state reported in EIA 861."
```

Comment:
This will display as "Hourly timeseries of estimated hourly electricity" -- is that intentional or one hourly too many?

Reply:
I'm happy with that justification!